Cartfile

```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 23.1.0-beta.2
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 116.0
-github "mapbox/mapbox-directions-swift" == 2.8.0-alpha.1
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 23.1.0-rc.1
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 117.0
+github "mapbox/mapbox-directions-swift" == 2.8.0-alpha.2
```

Comment:
Seems like Directions dependency wasn't updated for Carthage in previous release.